gulpfile.js

```diff
@@ -342,15 +332,38 @@ async function cookWithOptions(options = { formattingCheck: true }) {
 	return 0;
 }
 
+// Generates a regular expression to find references to the provided items
+function regularExpressionForFindingItemsInCache(cache) {
+
+    var regularExpressionSubstring = "";
+    let numberOfConditions = Object.keys(cache).length
+    var index = 0;
+    for (let itemId in cache) {
```

Comment:
I've started using `for of` since writing this code, thanks for bringing it up, I believe in this specific case this is correct as `cache` is not an array and `for of` actually causes an error. Probably a different way to iterate over the properties of an object (Object.something?)

Reply:
Error: ```TypeError: cache is not iterable```